ui/components/ui/button-group/index.scss

```diff
@@ -61,7 +61,7 @@
     }
 
     &:hover {
-      box-shadow: 0 0 14px rgba(0, 0, 0, 0.18);
+      box-shadow: var(--shadow-size-sm) var(--color-shadow-default);
     };
   }
 
```

Comment:
### Before

<img width="511" alt="button-group before1" src="https://user-images.githubusercontent.com/8112138/179422431-961764d3-631d-40a2-88dc-e40bc097ed70.png">
<img width="517" alt="button-group before2" src="https://user-images.githubusercontent.com/8112138/179422432-6453037a-b9c9-431c-814b-148985516c0e.png">

### After
<img width="532" alt="button-group after1" src="https://user-images.githubusercontent.com/8112138/179422441-af9824d4-545d-4283-9c64-a10259ebad47.png">
<img width="516" alt="button-group after2" src="https://user-images.githubusercontent.com/8112138/179422442-cddbb038-b401-46a7-aaf9-43bc21e0b96a.png">

Reply:
this shadow in SM looks good, I'm curious to know if we are applying it only in hover state or by default?